### ISSUE TITLE
Space shouldn't do cold damage

### ISSUE THREAD

**Issue by BellwetherLogic** [space-wizards/space-station-14#26478]
## Description
Space currently does cold damage. This is inaccurate. While the temperature of space is very low, it is also a vacuum, and vacuum is an extremely efficient _insulator_. If anything, crew should be overheating in spacesuits rather than taking cold damage. 

**Reproduction**
Go outside the station in an Emergency EVA suit with an O2 tank with an unmodified human and wait. You'll have hampered movement (beyond what the suit does) inside two minutes, be crit inside four, and dead in five. I tested this on Dev with a stopwatch.

**Additional context**
I am aware that the cold damage in space is there more to discourage casual forays outside the station in Emergency EVA suits than anything; however, cold is about the last thing that would kill someone in space, even in an uninsulated suit. Some quick googling seems to suggest that it would take a dead body (not a living one, mind, with ongoing homeostasis), anywhere from 18 to 36 hours to actually radiate enough heat to freeze in deep space. Even accounting for hypothermia in a living subject and a certain degree of abstraction when it comes to time in-game (since most shifts aren't 90 minutes long in real life, either), that's a far too quick time-to-crit.

In the spirit of constructive criticism, may I suggest that instead of making space cold, that the danger of an unprotected spacewalk instead be tied to the sun? Every station has solar panels and tracking of the sun is already a thing for them in-game; why not simply use the same code to generate a line-of-sight heat damage effect, since solar radiation is actually a significant issue in space. This could even do radiation damage during solar flare midround events, as the radioactive flux from the solar wind overpowers whatever radiation protection the station relies upon (stay away from external windows during solar flares!). If you wanted to be extra mean, you could even have line-of-sight to the sun cause eye damage without appropriate eye protection (or an EVA helmet). Making the sun be the threat is not only more accurate to real life, but has a degree of counterplay associated with it that could be very fun and will still serve to constrain options for unimpeded spacewalk. 

Finally, on a purely pedantic note: if they're not meant to be used outside the station, why is it called an Emergency Extra-Vehicular Activity (EVA) suit?

**Comment by IProduceWidgets** [comment]
I don't altogether hate the idea of space doing radiation damage instead of cold. It would definitely make radiation more of a prescient idea in crews mind instead of the "oops I guess your randomly dead" thing it is now.

**Comment by LordEclipse** [comment]
> I don't altogether hate the idea of space doing radiation damage instead of cold. It would definitely make radiation more of a prescient idea in crews mind instead of the "oops I guess youre randomly dead" thing it is now.

Yeah but then you'd need to grant hardsuits radiation resistance instead.. which opens up a slew of other balancing nightmares.

**Comment by EdenTheLiznerd** [comment]
Yeah, you're also likely to get mega cancer without a properly shielded suit to protect you from whatever the nearest star is throwing at you, a good way to go about this is up space damage by itself but remove the cold damage

**Comment by BellwetherLogic** [comment]
> I don't altogether hate the idea of space doing radiation damage instead of cold.

> Yeah but then you'd need to grant hardsuits radiation resistance instead.. which opens up a slew of other balancing nightmares.

This is exactly why I suggested using the sun as a source of low-but-constant heat damage (something under the structural damage limit), with the potential to add a radiation hazard specifically to the solar flare event. It lets radiation still be a potential hazard but keeps from every suit needing to have radiation resistance. Most suits are not intended to be used in space for long periods of time, so the odds that a flare's gonna happen while they're out there is pretty low, so why spend money on rad shielding?

**Comment by superjj18** [comment]
You can heal a cold corpse, you can't heal rad damage on a corpse

**Comment by BellwetherLogic** [comment]
> You can heal a cold corpse, you can't heal rad damage on a corpse

Which is why I don't think space should do a large amount of rad damage, or at least, not all the time. I think potential barotrauma as the primary threat plus a little heat damage and maybe a chance of eye damage/blindness when in line of sight to the sun without the appropriate equipment makes space a proper threat, and I think tying some extra rad damage to events like the solar flare make long-term spacewalking significantly more risky but not certain death. As the primary goal with making space more lethal is, as I understand it, to make space less attractive as a long-term solution to evading Sec, I think that fits the bill without making space an almost immediate death sentence.

**Comment by Kittygyat** [comment]
Closing this as-per triage procedure due to feature/balance requests not being valid issues without a cited maintainers prior conceptual approval.
Please feel free to consult a maintainer for conceptual approval before making balance changes or feature request issues next time.

Thank you for your issue report.
